plugins/com.google.cloud.tools.eclipse.appengine.libraries/META-INF/MANIFEST.MF

```diff
@@ -7,3 +7,6 @@ Bundle-Vendor: %providerName
 Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Bundle-ActivationPolicy: lazy
+Require-Bundle: org.eclipse.equinox.common
```

Comment:
It causes problems in this case: https://bugs.eclipse.org/bugs/show_bug.cgi?id=398886